docs/chapters/developer_guide.tex

```diff
@@ -0,0 +1,176 @@
+\chapter{Developer Guide}
+
+\section{Introduction}
+
+This chapter gives an overview of how \dfastmi can be setup for development.
+
+The development setup and configuration is done for usage of the visual studio code IDE. More information about visual studio code can be found on the internet.
+The development of python code project within Deltares is mostly done in with this tool. Visual Studio Code is free for private or commercial use. See the \href{https://code.visualstudio.com/license}{product license}) for details.
+
+\begin{Remark}
+	\item This configuration can not be used to build a standalone executable with nuitka tooling. It is for development purposes only.
+\end{Remark}
+
+\section{Configuration}
+The tool can be configured with a batch script which will configure using installed tools or ask the user to download and install the tools which can be used for them. The batch file is located in the sub folder \textbf{BuildScripts} and is called \textbf{DevelopDfastmi.bat}.
+
+The batch file will check for installed tooling for anaconda/miniconda. If not installed it will ask the user to install the anaconda/miniconda tooling for you. 
+\begin{Remark}
```

Comment:
```suggestion

\begin{Remark}
```

Reply:
Remark is now directly behind the sentence instead of a new line.

![fA0pkiK0n1](https://github.com/Deltares/D-FAST_Morphological_Impact/assets/102953403/aeb0973a-484b-4e60-acc0-5a84aa9f5920)